mysql-test/mroonga/storage/column/int/t/null.test

```diff
@@ -0,0 +1,41 @@
+# -*- mode: sql; sql-product: mysql -*-
+#
+# Copyright (C) 2024  Kodama Takuya <otegami@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+--source ../../../../include/mroonga/have_mroonga.inc
+--source ../../../../include/mroonga/load_mroonga_functions.inc
+
+--disable_warnings
+DROP TABLE IF EXISTS items;
+--enable_warnings
+
+CREATE TABLE items (
+  name VARCHAR(255),
+  price INT NULL,
+  KEY price_index(price)
+) DEFAULT CHARSET=utf8mb4;
+
+INSERT INTO items VALUES ("smile", 0);
+INSERT INTO items VALUES ("sample chips", NULL);
+
+SELECT mroonga_command("index_column_diff --table items#price_index --name index");
+
+SELECT * FROM items WHERE price = 0;
+
+DROP TABLE items;
+
+--source ../../../../include/mroonga/have_mroonga_deinit.inc
```

Comment:
```suggestion
--source ../../../../include/mroonga/unload_mroonga_functions.inc
--source ../../../../include/mroonga/have_mroonga_deinit.inc
```

Reply:
fix: [98bfd6c](https://github.com/mroonga/mroonga/pull/790/commits/98bfd6c96a77c761fcc80b177ef97e7a7f28f66f) Thanks you so much!